app/templates/index.html

```diff
@@ -34,7 +34,7 @@ <h1 class="page-title">All things tech at Columbia</h1>
             <h4>What We do</h4>
             <span class="bar yellow"></span>
         </div>
-        <h3>The Application Development Initiative seeks to nurture student creativity and technological aptitude by creating a community and resources for student developers of all talents and interests.</h3>
+        <h3>ADI is a student group at Columbia that aims to foster a community of students interested in technology.</h3>
```

Comment:
It seems weird that we have to repeat this. Is there a good way of factoring the description out into a Jinja2 macro or something?